src/components/NewComps/AffiliateForm/AffiliateForm.module.scss

```
@@ -0,0 +1,147 @@
+    overflow-x: hidden;
+    overflow-y: auto;
+    z-index: 10;
+    scrollbar-width: none;
+    -ms-overflow-style: none;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
+  }
+
+  :global(.iti__country-list--dropup) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)